pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>uk.gov.companieshouse</groupId>
+		<artifactId>companies-house-spring-boot-parent</artifactId>
```

Comment:
We should not be using spring boot parent pom anymore . Have a look at this slack conversation.
https://companieshouse.slack.com/archives/C0V5W57K4/p1594719953034600

Reply:
corrected